app/api/notes.js

```diff
@@ -8,6 +8,13 @@ function get(cb) {
 }
 
 function add(entry, cb) {
+  const length = entry.content.length;
+  if (length > 32) {
+    entry.title = `${entry.content.substring(0, 32)}...`;
+  } else {
+    entry.title = `${entry.content.substring(0, length)}...`;
```

Comment:
duplication of defining entry.title except for 32/length... perhaps a better way to write this?